contrib/test_cover.sh

```
@@ -1,9 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
-PKGS=$(go list ./... | grep -v '/simapp')
+PKGS=($(go list ./... | grep -v '/simapp'))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)